packages/marko-web-html-sitemap/marko.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Do you need this file? Pretty sure it's only required if you're defining components within the package.